orchestra/models.py

```diff
@@ -14,6 +12,56 @@
 # TODO(marcua): Convert ManyToManyFields to django-hstore referencefields or
 # wait for django-postgres ArrayFields in Django 1.8.
 
+class Workflow(models.Model):
+    """
+    Workflows describe the steps and requirements of completing expert work.
+
+    This model represents workflows that have been loaded into the database and
+    are usable by Orchestra projects. All workflows are also defined in the
+    codebase (see the `root_directory` attribute).
+
+    Attributes:
+        slug (str):
+            Unique identifier for the workflow.
+        name (str):
+            Human-readable name for the workflow.
+        description (str):
+            A longer description of the workflow.
+        code_directory (str):
+            The full path to the location of the workflow's manifest.
+    """
+    slug = models.CharField(max_length=200, unique=True)
+    name = models.CharField(max_length=200)
+    description = models.TextField()
+    code_directory = models.CharField(max_length=255, unique=True)
```

Comment:
Can you think of a scenario where we regret it? It would have to two different workflows needing to share the same code directory. I can't imagine this ever being a good idea, if we believe that workflows and their versions should be shared as Django apps.